docs/organizer-guide/meetups/livestreaming-meetups.rst

```diff
@@ -0,0 +1,40 @@
+
+We welcome streams or uploads to the Podcast YouTube channel, but first, we have to add you to the channel. A few days ahead of time, message @chrisward @tomjohnson or @jaredmorgs or the #podcast Slack channel with your email address and we will add you to the channel.
+
+With that access, you can upload a recorded meetup or live stream to the channel.
+
+Recording a Meetup
+------------------
+
+To record a meetup, all you need is a microphone. An external microphone is best, but the internal microphone on your laptop can work if you have nothing else.
+
+If you want video too, then again, an external camera is best, but positioned well, the internal camera in a laptop can work.
+
+If you want to mix the speaker and slides onto one recording, you can do this after the recording with video editing software, or live using software such as `Open Broadcast Studio (OBS) <https://obsproject.com/>`_.
+
+Once the recording is complete, you need to convert it to `a format YouTube supports <https://support.google.com/youtube/troubleshooter/2888402?hl=en>`_.
+
+Uploading a Recorded Meetup
+---------------------------
+
+Once you have access to the channel, you can upload a video to the channel as usual. Please add it to the "Meetup Recordings" playlist, and add as much detail to the description and metadata as you can.
+
+Streaming a Meetup
+------------------
+
+Livestreaming a meetup is harder, but once you have it set up, great for interacting live with remote participants. YouTube live streams even have a live chat for you to gather questions.
+
+You can start a live stream by clicking the "Create > Go Live" button in the YouTube top menu. You can schedule a live stream for a future date from the same button and then set a date in the following settings screen.
+
+Give your live stream a name, and then click "Create Stream." The next screen gives you a stream key that you need to connect to desktop (or mobile) streaming software. `Read this guide <https://support.google.com/youtube/answer/2907883?hl=en>`_ for a full list of what YouTube supports, and we stick with OBS for this guide.
+
+In the main OBS window, click "Settings > Stream," select "YouTube/YouTube Gaming," and paste the stream key into the text field below.
+
+Setup your audio and video inputs (which can include cameras, microphones, applications, and more) by reading `this OBS guide <https://obsproject.com/wiki/Sources-Guide>`_. It's a powerful application, and widely used, so you should be able to find lots of resources online to solve any issues you have. You can preview the stream your computer sends in the main preview window, and drag the video inputs to the locations and sizes you want. You can record and live stream at the same time by clicking both the "Start Streaming" and "Start Recording" buttons.
+
+When you're ready, click "Start Streaming," and you're live! Click "Start Streaming" again when you're finished to stop the stream.
+
```

Comment:
Maybe we can add a "contact us for any questions" section. Please use the support inbox for general questions, or your Slack handles for specific questions/problems.